### PR TITLE
Fdb test cases

### PR DESCRIPTION
**Pull request by richardyu-ms**
- Description of PR
Add Fdb test cases

- Approach
   - What is the motivation for this PR?
Add Fdb test cases

- How did you do it?
Test Group1: MAC learning
Case1: test_vlan_port_learn_disable
Case2: test_bg_port_learn_disable
Case3: test_non_bgPort_no_learn
Case4: test_new_vlan_member_learn
Case5: test_remove_vlan_member_no_learn
Case6: test_no_learn_invalidate_vlan
Case7: test_no_learn_broadcast_src
Case8: test_no_learn_multicast_src
Test Group2: FDB age
Case1: test_port_age
Case2: test_aging_after_move
Test Group3: FDB flush
Case1: test_flush_vlan_static
Case2: test_flush_vlan_dynamic
Case3: test_flush_port_static
Case4: test_flush_port_dynamic
Case5: test_flush_all_static
Case6: test_flush_all_dynamic
Case7: test_flush_all
Test Group4: MAC move
Case1: test_disable_move_drop
Case2: test_dynamic_mac_move
Case3: test_static_mac_move

- How did you verify/test it?
Test done:
Tested in PTF and DUT

### CODE REVIEW

File: test/sai_test/sai_fdb_test.py [hunk 1]
```diff
@@ -65,3 +67,1393 @@ def tearDown(self):
...
+        verify_packet(self, self.pkt, self.dev_port_list[2])
+
+        print("\tVerification complete")
+
+    def tearDown(self):
+        sai_thrift_remove_fdb_entry(self.client, self.fdb_entry1)
+        sai_thrift_remove_fdb_entry(self.client, self.fdb_entry2)
+        sai_thrift_remove_fdb_entry(self.client, self.moving_fdb_entry)
+        status = sai_thrift_flush_fdb_entries(self.client, entry_type=SAI_FDB_FLUSH_ENTRY_TYPE_DYNAMIC)
+        self.assertEqual(status, SAI_STATUS_SUCCESS)
```
Comment (ms-junyi) [inline test/sai_test/sai_fdb_test.py]: no blank line

File: test/sai_test/sai_fdb_test.py [hunk 2]
```diff
@@ -18,15 +18,16 @@
...
 #
 
+from unittest import skip
 from sai_test_base import T0TestBase
 from sai_thrift.sai_headers import *
 from ptf import config
 from ptf.testutils import *
 from ptf.thriftutils import *
 from sai_utils import *
+import pdb
```
Comment (ms-junyi) [inline test/sai_test/sai_fdb_test.py]: delete this line?

File: test/sai_test/sai_fdb_test.py (hunk 1 above)
Comment (richardyu-ms) [reply test/sai_test/sai_fdb_test.py]: those codes with the same purpose and within same logic, then put then in same block (without break lines)
https://peps.python.org/pep-0008/#blank-lines

File: test/sai_test/sai_fdb_test.py [hunk 3]
```diff
@@ -65,3 +67,1436 @@ def tearDown(self):
...
+        #Item: 15000950
+        attr = sai_thrift_get_switch_attribute(
+            self.client, available_fdb_entry=True)
+        self.assertEqual(attr["available_fdb_entry"] - stored_fdb_entry, 0)
+
+        send_packet(self, 2, self.chck_pkt)
+        verify_each_packet_on_multiple_port_lists(
+            self, [self.chck_pkt, self.chck_pkt], [self.dev_port_list[1:2], [self.dev_port_list[3:9]]])
+        verify_no_other_packets(self)
+        print("Verification complete")
```
Comment (Gfrom2016) [inline test/sai_test/sai_fdb_test.py:305]: Do we need to check fdb entry here?

**Review by Gfrom2016 — APPROVED** [review]

**Pull request merged**